public/app/plugins/datasource/graphite/MetricTankMetaInspector.tsx

```diff
@@ -11,48 +15,159 @@ export interface State {
-                <td>{row.retention} &nbsp;</td>
-                <td>{row.chunkspan} &nbsp;</td>
-                <td>{row.numchunks} &nbsp;</td>
-                <td>{row.ready} &nbsp;</td>
-              </tr>
-            ))}
-          </tbody>
-        </table>
-        <pre>{JSON.stringify(info, null, 2)}</pre>
+      <div className={styles.metaItem} key={key}>
+        <div className={styles.metaItemHeader}>Schema: {meta['schema-name']}</div>
+        <div className={styles.metaItemBody}>
+          <div className={styles.step}>
+            <div className={styles.stepHeading}>Step 1: Fetch</div>
+            <div className={styles.stepDescription}>
+              First data is fetched, either from raw data archive or a rollup archive
+            </div>
+
+            {rollupNotice && <p>{rollupNotice.text}</p>}
+            {!rollupNotice && <p>No rollup archive was used</p>}
+
+            <div>
+              {buckets.map((bucket, index) => {
+                const bucketLength = kbn.interval_to_seconds(bucket.retention);
+                const lengthPercent = (bucketLength / totalSeconds) * 100;
+                const isActive = index === meta['archive-read'];
+
+                return (
+                  <div key={bucket.retention} className={styles.bucket}>
+                    <div className={styles.bucketInterval}>{bucket.interval}</div>
+                    <div
+                      className={cx(styles.bucketRetention, { [styles.bucketRetentionActive]: isActive })}
+                      style={{ flexGrow: lengthPercent }}
+                    />
+                    <div style={{ flexGrow: 100 - lengthPercent }}>{bucket.retention}</div>
+                  </div>
+                );
+              })}
+            </div>
+          </div>
+
+          <div className={styles.step}>
+            <div className={styles.stepHeading}>Step 2: Normalization</div>
+            <div className={styles.stepDescription}>
+              Normalization happens when series with different intervals between points are combined.
+            </div>
+
+            {meta['aggnum-norm'] > 1 && <p>Normalization did occur using {normFunc}</p>}
+            {meta['aggnum-norm'] === 1 && <p>No normalization was needed</p>}
+          </div>
+
+          <div className={styles.step}>
+            <div className={styles.stepHeading}>Step 3: Runtime consolidation</div>
+            <div className={styles.stepDescription}>
+              If there are too many data points at this point Metrictank will consolidate them down to below max data
+              points (set in queries tab).
+            </div>
+
+            {runtimeNotice && <p>{runtimeNotice.text}</p>}
+            {!runtimeNotice && <p>No runtime consolidation</p>}
+          </div>
+        </div>
       </div>
     );
-  };
+  }
 
   render() {
     const { data } = this.props;
-    if (!data || !data.length) {
-      return <div>No Metadata</div>;
+
+    // away to dedupe them
```

Comment:
```suggestion
    // a way to dedupe them
```
my first typo suggestion ever!